src/cmap/establish/handshake/mod.rs

```diff
@@ -214,17 +211,12 @@ impl Handshaker {
     }
 
     /// Handshakes a connection.
-    pub(crate) async fn handshake(
-        &self,
-        conn: &mut Connection,
-        topology: Option<&Topology>,
-        handler: &Option<Arc<dyn SdamEventHandler>>,
```

Comment:
The monitors now handle emitting these events, so we don't need to pipe these event handlers down into the handshaker anymore.